app/controllers/external_protocols_controller.rb

```diff
@@ -64,14 +72,17 @@ def new
   # POST import_external_protocol
   def create
     service_call = ProtocolImporters::ImportProtocolService.call(
-      protocol_params: create_params[:protocol_params],
-      steps_params: create_params[:steps_paramas],
+      protocol_params: create_protocol_params,
+      steps_params: JSON.parse(create_steps_params[:steps]),
```

Comment:
We probably want to parse this inside the service to catch errors if parse fails here.